pyccel/codegen/printing/fcode.py

```diff
@@ -104,6 +104,9 @@
     'NumpyArcsinh': 'asinh',
     'NumpyArccosh': 'acosh',
     'NumpyArctanh': 'atanh',
+    'NumpyIsInfinte':'ieee_is_inf',
+    'NumpyIsFinite':'ieee_is_finite',
+    'NumpyIsNan':'ieee_is_nan',
```

Comment:
These elements have been added to the dictionary "numpy_ufunc_to_fortran", but `NumpyIsInfinite` etc, don't inherit from `NumpyUfuncBase`.

Reply:
Either they should be removed, or the inheritance should be fixed. I think the simplest and most logical thing is to fix the inheritance. These functions are indeed universal functions which operate on their elements one element at a time